custom/icds_reports/management/commands/backfill_supervisor_id.py

```diff
@@ -72,6 +72,8 @@ def get_sql_scripts(state_id):
         # some forms are from deleted locations, so loc table doesn't have data on some rows
         _table_name('static-usage_forms'),
         _table_name('static-child_tasks_cases'),
+        _table_name('static-ccs_record_cases_monthly_v2'),
```

Comment:
neither of these tables are in the dashboard anymore, are they being distributed?

Reply:
They are still large UCRs which have fairly high write load so it seems good to distribute them to me. They have always been marked for distribution on the citus tables sheet.